cfg/ack-1.0/master.yaml

```diff
@@ -896,25 +896,7 @@ groups:
           --root-ca-file=<path/to/file>
         scored: true
 
-      - id: 1.3.6
```

Comment:
Rather than removing the check and disrupting the sequence, which may confuse the user, remove 'scored' field and set the type to "skip." If it is set to "skip," it will be marked as "INFO.".

Reply:
Thanks. Taken care of in new PR - https://github.com/aquasecurity/kube-bench/pull/1685